examples/iam-read-only-policy/versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.15.0"
+
+  required_providers {
+    aws = ">= 2.23"
```

Comment:
If I remember correctly, 2.23 does not work with Terraform 0.15.

Please downgrade the minimum version of Terraform (see comment above).

Reply:
done